### ISSUE TITLE
All children elements of medium-editor can be permanently deleted, causing the editor to stop working

### ISSUE THREAD

**Issue by jsindos** [orthes/medium-editor-insert-plugin#408]
**TL;DR**
medium-editor-insert-plugin `v2.4.0` and medium-editor `v5.22.1` has a bug which isn't present when using medium-editor `v5.10.0`.

Playing around with the example in medium-editor-insert-plugin `v2.4.0` and medium-editor `v5.22.1` causes no new `<p>` tag to be spawned when the last `<p>` of the editor is backspaced, giving the following DOM structure:

![screen shot 2017-01-07 at 1 08 38 pm](https://cloud.githubusercontent.com/assets/8110172/21738558/0e32505e-d4db-11e6-81a5-c23ac908c06a.png)

Which looks like:

![screen shot 2017-01-07 at 1 06 09 pm](https://cloud.githubusercontent.com/assets/8110172/21738560/141a3cde-d4db-11e6-90ec-0bc898ee9678.png)

(No blinking cursor as there are no `<p>` children.)

This bug does not occur on the [official hosted example](http://linkesch.com/medium-editor-insert-plugin/), running medium-editor `v5.10.0` + medium-editor-insert-plugin `v2.2.3`.

**SOLUTION**
`v5.22.0` of medium-editor introduced a change to how they handle removal of first empty paragraph on backspace: https://github.com/yabwe/medium-editor/pull/1187

```
} else if (MediumEditor.util.isKey(event, MediumEditor.util.keyCode.BACKSPACE) &&
        MediumEditor.util.isMediumEditorElement(node.parentElement) &&
        !node.previousElementSibling &&
        node.nextElementSibling &&
        isEmpty.test(node.innerHTML)) {

    // when cursor is in the first element, it's empty and user presses backspace,
    // do delete action instead to get rid of the first element and move caret to 2nd
    event.preventDefault();
    MediumEditor.selection.moveCursor(this.options.ownerDocument, node.nextSibling);
    node.parentElement.removeChild(node);
}
```

As there is a `nextElementSibling` (`medium-insert-buttons`) this check passes and the `<p>` is deleted.

**Comment by dreamapplehappy** [comment]
you can use the code below to prevent this problem
```
let that = this;
this.editor.subscribe('editableKeyup', function (e, editable) {
                // watch the content and solve this problem
                let allContents = that.editor.serialize();
                let content = allContents["element-0"].value;
                if(!content) {
                    that.editor.setContent('<p><br></p>');
                }
});
```
this works for me 😂

**Comment by zengxiaoluan** [comment]
@dreamapplehappy 属性“element-0”访问不到，改成了
`getRealContents: function (contents) {
    for (var i in contents) {
         return contents[i].value
                    }
                }`
